core/src/main/scala/org/apache/spark/SSLOptions.scala

```diff
@@ -174,6 +177,25 @@ private[spark] object SSLOptions extends Logging {
   def parse(conf: SparkConf, ns: String, defaults: Option[SSLOptions] = None): SSLOptions = {
     val enabled = conf.getBoolean(s"$ns.enabled", defaultValue = defaults.exists(_.enabled))
 
+    def temporaryStoreFile(b64: String): File = {
+      val bytes = DatatypeConverter.parseBase64Binary(b64)
+      val storeFile = Files.createTempFile("store", "tmp",
+        PosixFilePermissions.asFileAttribute(PosixFilePermissions.fromString("rw-------")))
+      storeFile.toFile.deleteOnExit() // just to be sure
+      Files.write(storeFile, bytes)
+      storeFile.toFile
+    }
+
+    conf.getOption(s"$ns.keyStoreBase64").foreach(b64 => {
```

Comment:
I know this is going to be internal spark features, but still be good if we can document every conf we added so we can point our customers to them. Look at docs/running-on-mesos and configuration.md